calico_cni_k8s_test.go

```diff
@@ -602,6 +607,137 @@ var _ = Describe("CalicoCni", func() {
 					Expect(pod2IP).Should(Equal(expectedIP))
 				})
 			})
+
+			Context("ADD a continaer with a ContainerID and DEL it with the same ContainerID then ADD a new container with a different ContainerID, and send a DEL for the old ContainerID", func() {
```

Comment:
Add a comment with a link to the specific issues so we know why this test was added